modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/RosettaSearch.scala

```diff
@@ -0,0 +1,20 @@
+package org.tessellation.rosetta.server.model
+
+import org.tessellation.schema.transaction.Transaction
+
+object RosettaSearch {
+
+  def searchBySourceAddress(transactionHash: String, snapshots: List[Transaction]) =
+    snapshots.filter(_.source == transactionHash)
+
+  def searchByDestinationAddress(transactionHash: String, snapshots: List[Transaction]) =
+    snapshots.filter(_.destination == transactionHash)
+
+  def searchByAddress(transactionHash: String, snapshots: List[Transaction]) = {
+    val sourceTransactions = searchBySourceAddress(transactionHash, snapshots)
+    val destinationTransactions = searchByDestinationAddress(transactionHash, snapshots)
+
+    sourceTransactions ++ destinationTransactions
+  }
```

Comment:
why it's in the model? also, argument `snapshot` is of type `List[Transaction]` which is wrong.

Reply:
This object is unneeded. Removed.